mlflow/models/model.py

```diff
@@ -1109,6 +1117,22 @@ def log_model_metrics_for_step(client, model_id, run_id, step):
             if registered_model is not None:
                 model_info.registered_model_version = registered_model.version
 
+                # Print a link to the UC model version page if the model is in UC.
+                registry_uri = mlflow.get_registry_uri()
+                workspace_url = get_workspace_url()
+                if workspace_url is not None and is_databricks_unity_catalog_uri(registry_uri):
```

Comment:
```suggestion
                if is_databricks_unity_catalog_uri(registry_uri) and (url := get_workspace_url()):
```

to skip `get_workspace_url` when `is_databricks_unity_catalog_uri(registry_uri)` is false.